scripts/ci/test_plan.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
 # SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 NXP
```

Comment:
you need (c) here

Reply:
because (c) means "Copyright", so NXP is using it

Reply:
the rationales it that `(c)` means "Copyright", and having "Copyright (c)" is redundant, so the preferred way according to the company policy is to use "Copyright <year> NXP" (see for instance https://github.com/zephyrproject-rtos/hal_nxp/blob/master/dts/nxp/nxp_imx/mimx6sx-pinctrl.dtsi#L2). Older copyrights from NXP may include "Copyright (c)".